modules/system/console/WinterUtil.php

```diff
@@ -204,6 +204,12 @@ protected function utilCompileLang()
             $overridePath = base_path() . '/lang/'.$locale.'/system/client.php';
             if (File::isFile($overridePath)) {
                 $messages = array_replace_recursive($messages, require $overridePath);
+            } elseif (str_contains($locale, '-')) {
+                list($parentLocale, $country) = explode('-', $locale);
+                $overridePath = base_path() . '/lang/'.$parentLocale.'/system/client.php';
+                if (File::isFile($overridePath)) {
+                    $messages = array_replace_recursive($messages, require $overridePath);
+                }
```

Comment:
This probably shouldn't be an elseif because I would assume that client overrides behave like any other override by respecting the fallback language. 

I.e. if I have `/lang/en/system/client.php` and `/lang/en-ca/system/client.php`, I would expect it to merge all of the messages such that `array_replace_recursive($original, $parentLocale, $childLocale)` with appropriate handling for if a child locale exists but the parent doesn't and vice versa.

Reply:
Good point @LukeTowers, I'll make the change!